tools/pipeline/pipe/Main.cpp

```diff
@@ -115,7 +115,10 @@ int main(int argc, char *argv[]) {
     AbortOnError(Path.check());
     AbortOnError(Set.at(DefaultNames[I]).load(Path));
   }
+
   auto Enumeration = Set.enumerate();
+  ClonedPipe.Pipe->deduceResults(Manager.context(), Enumeration);
```

Comment:
So that was it? I'm impressed `:P`

Please better explain in the commit message what this commit does and why.